dropwizard-jackson/src/main/java/io/dropwizard/jackson/Jackson.java

```diff
@@ -64,6 +65,9 @@ private static ObjectMapper configure(ObjectMapper mapper) {
         if (JavaVersion.isJava8()) {
             mapper.registerModule(new AfterburnerModule());
         }
+        if (JavaVersion.isJava11OrHigher()) {
+            mapper.registerModule(new BlackbirdModule());
+        }
```

Comment:
Afterburner and Blackbird are incompatible, so we should only register one of them.
```suggestion
        if (JavaVersion.isJava11OrHigher()) {
            mapper.registerModule(new BlackbirdModule());
        } else if (JavaVersion.isJava8()) {
            mapper.registerModule(new AfterburnerModule());
        }
```